waspc/src/Wasp/Psl/Parser/Model.hs

```diff
@@ -52,7 +47,7 @@ model = do
 -- which is everything besides model keyword, name and braces:
 --   `model User { <body> }`.
 body :: Parser Psl.Model.Body
-body = Psl.Model.Body <$> many1 element
+body = Psl.Model.Body <$> some element
```

Comment:
Aha `some` is alternative to `many1`, ok makes sense. I think `many1` requirest at least one element, but I guess so does `some` or it would be named `any` if it didn't :D.

Reply:
Yep, `many` for 0 or more and `some` for 1 or more.